frontend/src/scenes/persons/PersonFeedCanvas.tsx

```diff
@@ -16,11 +16,21 @@ const PersonFeedCanvas = ({ person }: PersonFeedCanvasProps): JSX.Element => {
     const id = person.id
     const distinctId = person.distinct_ids[0]
 
+    const personFilter: AnyPropertyFilter[] = [
+        {
+            type: PropertyFilterType.EventMetadata,
+            key: 'person_id',
+            value: id,
+            operator: PropertyOperator.Exact,
+        },
+    ]
+
     return (
         <Notebook
             editable={false}
             shortId={`canvas-${id}`}
             mode="canvas"
+            canvasFiltersOverride={personFilter}
```

Comment:
This feels like a very specific prop to be adding to a generic component. There's possibly two options here:

- Why not wrap the `Notebook` in `<BindLogic logic={personFeedCanvasLogic} props={personFilter}>` (or something similar) and check to see if the logic exists within the `NotebookNodeQuery`
- If you want to continue passing the props in like this then maybe better to do something like `context={{ overrideFilters: personFilters }}`